### PR TITLE
Add WiCS/WiM Dinner with the Professors event post

### PR DESCRIPTION
**Pull request by taliamccormick**

### CODE REVIEW

File: content/W2017/W17-event-dinner-with-professors.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+title: Dinner with the Professors
+Date: 2017-03-1
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Student-faculty dinner.
+
+Come have dinner with the brilliant women who are professors at Waterloo! This is a wonderful opportunity to learn about their research, academia, teaching, and your own academic interests.
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md]: Lines need to be 80 char long.

File: content/W2017/W17-event-dinner-with-professors.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Note: This event is women-only and requires registration.
+
+## Event Details ##
+
++ **Who:** [Cis & trans women and non-binary individuals]({filename}/pages/faq.md)
++ **What:** Student-faculty dinner
++ **Where:** University Club
++ **When:** March 8, 2017 6:30PM
++ **Note:** Tickets for students are sold for $10 in MathSoc, until Wednesday, March 1. Sign-up for professors and for grad students who are interested in attending as mentors is free and run through eventbrite (here: goo.gl/B8PQHL ). Registration is required and is first-come first-serve. If you can not make it to the MathSoc office in-person, please email us at wics-ugrad@lists.uwaterloo.ca to find a workaround.
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md]: This note should go in the main part of the post. The lines also need to be 80 char long. 

The link to the eventbrite should be embedded. i.e [evenbrite](goo.gl/B8PQHL). 

"Run through eventbrite" -> "registrations is available [here](goo.gl/B8PQHL)"

File: content/W2017/W17-event-dinner-with-professors.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+title: Dinner with the Professors
+Date: 2017-03-1
+Category: Events
+Tags: events, social
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Student-faculty dinner.
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md]: This should be something like "Join WiCS for our second dinner with profs event!" or something to that effect. i.e more pep

**Review by annalorimer — CHANGES_REQUESTED** [review]

**Review by fat1996 — CHANGES_REQUESTED** [review]
it would be better to italicize or underline the wics email address.

**Comment by taliamccormick** [comment]
Looks better?
Should I hyperlink the email address, also?

**Comment by fat1996** [comment]
@taliamccormick yeah, that'll be better.

File: content/W2017/W17-event-dinner-with-professors.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Summary: Join WiCS for our second Dinner with the Professors event!
+
+Come have dinner with the brilliant women who are professors at Waterloo! This
+is a wonderful opportunity to learn about their research, academia, teaching,
+and your own academic interests. 
+
+Note: Tickets for students are sold for $10 in MathSoc, until Wednesday, March
+1. Registration for professors and for grad students who are interested in
+attending as mentors is free and available [here](goo.gl/B8PQHL). Registration
+is required and is first-come first-serve. If you can   not make it to the
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md]: Extra spaces between "can" and "not"

File: content/W2017/W17-event-dinner-with-professors.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+Slug: dinner-with-professors-W17
+Author: Talia McCormick
+Summary: Join WiCS for our second Dinner with the Professors event!
+
+Come have dinner with the brilliant women who are professors at Waterloo! This
+is a wonderful opportunity to learn about their research, academia, teaching,
+and your own academic interests. 
+
+Note: Tickets for students are sold for $10 in MathSoc, until Wednesday, March
+1. Registration for professors and for grad students who are interested in
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md:14]: This miiiiiight render as a numbered list. Not sure, heroku is being stupid.

File: content/W2017/W17-event-dinner-with-professors.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+attending as mentors is free and available [here](goo.gl/B8PQHL). Registration
+is required and is first-come first-serve. If you can   not make it to the
+MathSoc office in-person, please email us at [*wics-ugrad@lists.uwaterloo.ca*](mailto:wics-ugrad@lists.uwaterloo.ca) to
+find   a workaround.
```
Comment (annalorimer) [inline content/W2017/W17-event-dinner-with-professors.md]: Extra spaces between "find" and "a"

**Review by annalorimer — CHANGES_REQUESTED** [review]

File: content/W2017/W17-event-dinner-with-professors.md (hunk 5 above)
Comment (taliamccormick) [reply content/W2017/W17-event-dinner-with-professors.md:14]: It looks fine when I run locally.
screenshot 2017-03-02 22 36 39
Putting the "1." back a line will make the previous line greater than 80 characters wide. I could change it to say "March first" instead. Alternatively, I could move March onto the next line, also.

Comment (annalorimer) [reply content/W2017/W17-event-dinner-with-professors.md:14]: Perfect.

**Review by annalorimer — APPROVED** [review]

**Pull request merged**